subscriptions/api.go

```diff
@@ -0,0 +1,21 @@
+package subscriptions
+
+import (
+	zoho "github.com/schmorrison/Zoho"
+)
+
+const ZohoSubscriptionsOriganizationID = "X-com-zoho-subscriptions-organizationid"
+
+// API is used for interacting with the Zoho Subscriptions API
+type API struct {
+	*zoho.Zoho
+	OrganizationID string
+}
+
+// New returns a *subscriptions.API with the provided zoho.Zoho as an embedded field
+func New(z *zoho.Zoho, organizationID string) *API {
+	return &API{
+		Zoho:           z,
+		OrganizationID: organizationID,
+	}
+}
```

Comment:
Even though it probably isn't used by anyone, I would like to see the ID generated/added to the `API` struct. Just to keep thing consistent unless there is a reason to not do that.

Reply:
added id generation same way as in invoice package